src/prop-types.js

```diff
@@ -13,10 +13,11 @@ import {
   ACTIVITY_STATUS_FAILED,
   ACTIVITY_STATUS_PENDING,
   ACTIVITY_STATUS_TIMED_OUT,
-  TRANSACTION_STATUS_ERROR,
+  TRANSACTION_STATUS_UPCOMING,
+  TRANSACTION_STATUS_SIGNING,
   TRANSACTION_STATUS_PENDING,
   TRANSACTION_STATUS_SUCCESS,
-  TRANSACTION_STATUS_UPCOMING,
+  TRANSACTION_STATUS_ERROR,
```

Comment:
I don't think this change is relevant for the purpose of the PR, but if there is a good reason for this, I'd love some help to understand it

Reply:
See my comment to your similar question at https://github.com/aragon/aragon/pull/1210/files/b795801d8d634b00462de36a82f9c4fa414b178c#r362507292